cmd/sqlflowserver/main_test.go

```diff
@@ -1088,6 +1089,34 @@ INTO sqlflow_models.my_xgb_regression_model;
 	ParseRow(stream)
 }
 
+// CaseAnalyzeXGBoostModel is used to test analyze a xgboost model
+func CaseAnalyzeXGBoostModel(t *testing.T) {
+	a := assert.New(t)
+	stmt := `
+SELECT *
+FROM housing.train
+ANALYZE sqlflow_models.my_xgb_regression_model
```

Comment:
Please note that go test may run test cases unordered, the model may be not generated when running this test, it's better to generate this model first before running `ANALYZE`.

Reply:
The [ref](https://github.com/golang/go/blob/a813d3c788b4ec58032616e8d269ee65d1b10085/src/testing/testing.go#L791-L792) introduces a `go test` run parallel with other tests if and only if the `t.Parallel()` was called.

Reply:
This is not about parallelizm, it's about the order, see: https://stackoverflow.com/questions/31201858/how-to-run-golang-tests-sequentially, it says:

> You can't / shouldn't rely on test execution order. The order in which tests are executed is not defined, and with the use of testing flags it is possible to exclude tests from running, so you have no guarantee that they will run at all.

Reply:
In our e2e test, the [subtests](https://blog.golang.org/subtests) are ordered by group
https://github.com/sql-machine-learning/sqlflow/blob/672d4ef75fcaa955ddab1fc5f03278d51c77a4b4/cmd/sqlflowserver/main_test.go#L308-L310
But we would run `go test -run CaseTrainAndAnalyzeXGBoostModel`  independently some times, it needs an existed model, so I add the several codes for that.